collect_app/src/androidTest/java/org/odk/collect/android/formentry/FieldListUpdateTest.java

```diff
@@ -378,6 +379,19 @@ public void manuallySelectingAValueForMissingExternalApp_ShouldTriggerUpdate() {
         onView(withText("Target14")).check(matches(isDisplayed()));
     }
 
+    @Test
+    public void search_function_in_field_list() throws InterruptedException {
+        jumpToGroupWithText("Search in field-list");
+        onView(withText(startsWith("Source15"))).perform(click());
+        Thread.sleep(1000);
```

Comment:
Why are this sleep and the `InterrputedException` throwing needed? Espresso should be able to detect when the UI thread is idle. Is it related to database lookups? `ExternalCsvSearchTest` does something similar so I'm surprised.

Reply:
It's because the spinner is not clickable immediately and the test didn't work.